src/core/blockchain/ethereum/config.ts

```diff
@@ -30,6 +30,7 @@ export const config: IBlockchainConfig = {
     coin: 'ETH',
     defaultUnit: 'WEI',
     iconComponent: EthIcon,
+    nrBlocksPassed: 10,
```

Comment:
we need to do some research here, i remember on ethereum we can consider a transaction dropped after 50 blocks